asn.go

```diff
@@ -0,0 +1,25 @@
+package asnutil
+
+import (
+	"net"
+
+	"github.com/libp2p/go-libp2p-asn-util/trie"
+)
+
+var Store *store
+
+func init() {
+	m, err := trie.UnmarshalRanges([]byte(cidrASNRaw))
+	if err != nil {
+		panic(err)
+	}
+	Store = &store{m}
```

Comment:
I suspect we're still better off with an indirect store here where we unmarshal either on first use or in a goroutine.